File: 6TH_SEMESTER/SWE 4604 - SOFTWARE TESTING AND QUALITY ASSURANCE LAB/6/testinglab06/src/tests/ContactPage.java
package tests;

import org.junit.jupiter.api.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

//210042112

public class ContactPage extends BaseTest{
    @Test
    public void ContactSubmit()
    {
        driver.get("https://practicesoftwaretesting.com/contact");
        WebElement firstname = driver.findElement(By.id("first_name"));
        firstname.sendKeys("Namisa");

        WebElement lastname = driver.findElement(By.id("last_name"));
        lastname.sendKeys("Najah");

        WebElement email = driver.findElement(By.id("email"));
        email.sendKeys("namisa.najah.raisa@gmail.com");


        WebElement dropdown = driver.findElement(By.id("subject"));
        dropdown.click();

        WebElement option = driver.findElement(By.cssSelector("option[value='warranty']"));
        option.click();

        WebElement messagebox = driver.findElement(By.id("message"));
        messagebox.sendKeys("This is a test message.");
        WebElement submitting = driver.findElement(By.className("btnSubmit"));

        submitting.click();
    }
}
// -_- N4M154 -_-

File: 6TH_SEMESTER/SWE 4604 - SOFTWARE TESTING AND QUALITY ASSURANCE LAB/6/Lab06CSharp/TestProject1/ContactPage.cs
using OpenQA.Selenium;

namespace Lab06
{
    public class ContactPage : BaseTest
    {
        [Fact]
        public void ContactSubmit()
        {
            driver.Navigate().GoToUrl("https://practicesoftwaretesting.com/contact");

            var firstName = driver.FindElement(By.Id("first_name"));
            firstName.SendKeys("Namisa");

            var lastName = driver.FindElement(By.Id("last_name"));
            lastName.SendKeys("Najah");

            var email = driver.FindElement(By.Id("email"));
            email.SendKeys("namisa.najah.raisa@gmail.com");

            var subjectDropdown = driver.FindElement(By.Id("subject"));
            subjectDropdown.Click();

            var warrantyOption = driver.FindElement(By.CssSelector("option[value='warranty']"));
            warrantyOption.Click();

            var messageBox = driver.FindElement(By.Id("message"));
            messageBox.SendKeys("This is a test message.");

            var submitButton = driver.FindElement(By.ClassName("btnSubmit"));
            submitButton.Click();

        }
    }
}
